dist/images/daemonset.sh

```diff
@@ -64,6 +64,7 @@ OVN_EGRESSIP_HEALTHCHECK_PORT=
 OVN_EGRESSFIREWALL_ENABLE=
 OVN_EGRESSQOS_ENABLE=
 OVN_DISABLE_OVN_IFACE_ID_VER="false"
+OVN_MULTI_NETWORK_ENABLE=
```

Comment:
> I also think it would be better if this last commit got moved to a separate PR.

sure.

> another with the network-attach-def controller.
> the third with everything related to implementing the secondary networks for the routed topology.

so the one commit which introduce network-attach-def controller will just implement the  multihoming controller without handling any events?

Reply:
IMO, yes. That would give us the insight into how the controller is being setup, which, at this stage, I think is the interesting part.

@jcaamano / @trozet  ?